source/Generic/PlayState/PlayState.cs

```diff
@@ -392,6 +425,175 @@ private void WindowClosed(object sender, EventArgs e)
+            {
+                if (playtime.Seconds == 1)
+                {
+                    return String.Format(ResourceProvider.GetString("LOCPlayState_SecondPlayed"), playtime.Seconds.ToString());
+                }
+                else
+                {
+                    return String.Format(ResourceProvider.GetString("LOCPlayState_SecondsPlayed"), playtime.Seconds.ToString());
+                }
+            }
+            else
+            {
+                if (playtime.Minutes == 1)
+                {
+                    return String.Format(ResourceProvider.GetString("LOCPlayState_HoursMinutePlayed"), playtimeHours.ToString(), playtime.Minutes.ToString());
+                }
+                else
+                {
+                    return String.Format(ResourceProvider.GetString("LOCPlayState_HoursMinutesPlayed"), playtimeHours.ToString(), playtime.Minutes.ToString());
+                }
+            }
+        }
+
+        /// <summary>
+        /// Method for showing notifications. It will respect the style (Playnite / Windows) notification settings.<br/><br/>
+        /// <param name="status">Status of the game to be notified:<br/>
+        /// - "resumed" for resuming process and playtime<br/>
+        /// - "playtimeResumed" for resuming playtime<br/>
+        /// - "suspended" for suspend process and playtime<br/>
+        /// - "playtimeSuspended" for suspend playtime<br/>
+        /// - "information" for showing the actual status<br/>
+        /// </param>
+        /// </summary>
+        private void ShowNotification(string status)
+        {
+            switch (status)
+            {
+                case "resumed": // for resuming process and playtime
+                    if (!showWindowsNotificationsStyle)
+                    {
+                        ShowSplashWindow(ResourceProvider.GetString("LOCPlayState_StatusResumedMessage"));
+                    }
+                    else
+                    {
+                        // Windows notification only admits 3 .Addtext properties, so for more text lines you have to use {Environment.NewLine}
+                        new ToastContentBuilder()
+                            .AddText(currentGame.Name) // First AddText field will act as a title
+                            .AddText(ResourceProvider.GetString("LOCPlayState_StatusResumedMessage"))
+                            .AddText($"{ResourceProvider.GetString("LOCPlayState_Playtime")} {GetHoursString(GetRealPlaytime())}" +
+                                $"{Environment.NewLine}{ResourceProvider.GetString("LOCPlayState_TotalPlaytime")} {GetHoursString(GetRealPlaytime() + currentGame.Playtime)}")
+                            .AddHeroImage(new Uri(Path.Combine(PlayniteApi.Paths.ConfigurationPath, "library/files", currentGame.BackgroundImage))) // Show game image in the notification
```

Comment:
Don't combine the path here. Use this method instead https://playnite.link/docs/master/api/Playnite.SDK.IGameDatabaseAPI.html#Playnite_SDK_IGameDatabaseAPI_GetFullFilePath_System_String_

Example: https://playnite.link/docs/master/tutorials/extensions/library.html#exporting-game-cover

Also, are the toasts compatible with urls? In case it doesn't, then that needs to be handled in case the game has an url set for the image

Reply:
I checked but didn't find a better way, thanks for pointing me out.

The toasts are compatible with links, in their doc they have this example:
`.AddInlineImage(new Uri("https://picsum.photos/360/202?image=883"))`

https://docs.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/send-local-toast?tabs=uwp

Reply:
Done